src/components/user/IFXUser.js

```diff
@@ -365,7 +365,7 @@ export class User extends IFXItemBase {
   }
 
   hasGroup(group) {
-    return group in this.groups
```

Comment:
Does this not work? Or does it not work if `groups` is `undefined`?

Reply:
Did not work